src/scenes/AcceleratorRouter/Cohorts/CohortsListView.tsx

```diff
@@ -46,12 +46,15 @@ const columns = (activeLocale: string | null): TableColumnType[] =>
       ]
     : [];
 
-const CohortsTable = ({ filterModifiers, extraTableFilters }: CohortTableProps) => {
+const CohortsListView = ({ filterModifiers, extraTableFilters }: CohortsListViewProps) => {
   const dispatch = useAppDispatch();
   const { activeLocale } = useLocalization();
   const history = useHistory();
+  const { isAllowed } = useUser();
+  const { isMobile } = useDeviceInfo();
 
   const cohorts = useAppSelector(selectCohorts);
+  const isEmptyState = !cohorts?.length;
```

Comment:
We probably don't want to flash the empty state while the search is happening.
Also don't want to show the empty state if search or filters ends up returning empty results.
Suggestion:
`const isEmptyState = cohorts?.length === 0 && noSearchAndNoFilters;`
There should be a way to find out if there are search params. The Participants list view does something like this, if you'd like a reference.

Reply:
Good call 👍